src/chrome/content/rules/TouTiao.xml

```diff
@@ -2,36 +2,48 @@
 	404:
 		space.toutiao.com	(Redirect to 404error page)
 
+	Mixcontented:
+		www.toutiao.com
+		http: mode couse MCB issue. Test in http://www.toutiao.com/i6371298813642015233/
+
 	MCB:
-		^(www.)?toutiao.com	( with videos )
 		app.toutiao.com
 		m.toutiao.com
 
 	Mismatch:
 		(www.)?pstatp.com
 		i.pstatp.com
 		m.pstatp.com
-		p(0|1|2|4|5a?|6|7|8|9|10).pstatp.com	( equal to p3 )
+		v(0|1|2|4|5|6|7|9|10).pstatp.com
 		s[4-5].pstatp.com
 		(p|v)[0-10].statp.com
 -->
 
-<ruleset name="toutiao.com (partial)">
+<ruleset name="TouTiao (partial)">
+
+	<!-- Complications: -->
+	<target host="s4.pstatp.com" />
+	<target host="s5.pstatp.com" />
+	<rule from="^http://s(4|5)\.pstatp\.com/" to="https://s6.pstatp.com/" />
+		<test url="http://s4.pstatp.com/toutiao/resource/ntoutiao_web/static/image/logo_201f80d.png" />
+		<test url="http://s5.pstatp.com/toutiao/resource/ntoutiao_web/static/image/logo_201f80d.png" />
 
-	<!-- Complication: -->
+	<!-- Directly: -->
 	<target host="toutiao.com" />
 	<target host="www.toutiao.com" />
-		<test url="http://toutiao.com/a6361374475379998978/" />
-		<test url="http://toutiao.com/i6360544217684509186/" />
-		<test url="http://toutiao.com/i6361599096398021121/" />
 		<test url="http://www.toutiao.com/a6361374475379998978/" />
 		<test url="http://www.toutiao.com/i6360544217684509186/" />
-		<test url="http://www.toutiao.com/i6361599096398021121/" />
-	<exclusion pattern="^http://(www\.)?toutiao\.com/(a|i)(\d+)/" />
```

Comment:
I'm a bit confused about this change: You removed the exclusion but referring to your comments there are still mixed content issues?